### ISSUE TITLE
Maven's artifact contains Expr generated code from the example 

### ISSUE THREAD

**Issue by monxalo** [mike-lischke/antlr4-c3#10]
Checking external libraries on my IDE, it has these files:

```
com.vmware.antlr4c3
- CodeCompletionCore
- ExprBaseListener
- ExprBaseVisitor
- ExprLexer
- ExprListener
- ExprParser
- ExprVisitor
```

**Comment by mike-lischke** [comment]
Can you be more specific what this issue is about?

**Comment by monxalo** [comment]
Sorry if wasn't clear enough.

The generated code from the java port (ExprLexer, ExprListener) is being included on maven's jar artifact: `antlr4c3`. It should only contain the `CodeCompletionCore`, right?

**Comment by nick-stephen** [comment]
You're right, thanks for pointing this out, I'll fix it.

**Comment by nick-stephen** [comment]
... have just fixed and re-uploaded 1.0.0 release without these test classes. No change to production code, so didn't bump the release number. Will do next time :-)
